.github/workflows/create_release.yml

```diff
@@ -1,13 +1,12 @@
-# yamllint --format github .github/workflows/create_release.yml
```

Comment:
can you pull the workflow updates, and the version bumps in docker and pom.xml into a PR, then rebase this? I would like to see only stuff about the encoder side and review that. cheers!